app/assets/javascripts/admin/templates/badges-award.hbs

```diff
@@ -0,0 +1,22 @@
+{{#d-section class="award-badge"}}
+  <form class="form-horizontal">
+    <h1>{{i18n 'admin.badges.mass_award.title'}}</h1>
+    <div class='badge-preview'>
+      {{#if model}}
+        {{icon-or-image model}}
```

Comment:
I noticed in the screenshot that this is extremely big. Could we shrink it a bit?